### DISCUSSION TITLE
uBlock origin blocking self hosted plausible

### DISCUSSION THREAD

**Discussion by spapas** [plausible/analytics#2481] [Anything Else]
Hello, I'm self hosting plausible on a domain named https://plausible.example.com and I see that when I load a page that has the plausible script it tried to retieve 

`https://plausible.example.com/js/plausible.js` and it is blocked by ublock origin.

If I disable the ublock origin it works fine.

I'd like to ask if anybody knows a way to fool ublock origin so it won't block plausible.js. For example, what happens if I changed the hostname of plausible.js to `https://goodserver.example.com/js/goodfile.js` ? Would then ublock origin download it?  Has anybody tried that?

**Comment by susanthenerd** [comment]
Hello, please read this first: https://plausible.io/docs/proxy/introduction.
Simply said, uBlock Origin is not blocking the domain, but everything that has "/js/plausible.js" at the end.

**Comment by spapas** [comment]
Thank you very much for the answer! I'll try renaming the file to see what happens. Do you know if this can be done by the plausible build system somehow or I need to do it manually?

**Reply by susanthenerd** [reply]
It is more recommended to proxy it.
Editing the file name will just add the same issues each time plausible is going to be updated. It should be possible to do it in the build system, but it will still generate issues when updating.

**Reply by spapas** [reply]
That's a really great idea thank you!
